src/System.Drawing.Common/src/misc/DbgUtil.cs

```diff
@@ -17,7 +17,7 @@ internal sealed class DbgUtil
         public static void AssertFinalization(object obj, bool disposing)
         {
 #if GDI_FINALIZATION_WATCH
-            if( disposing || AppDomain.CurrentDomain.IsFinalizingForUnload() )
+            if ( disposing || AppDomain.CurrentDomain.IsFinalizingForUnload() )
```

Comment:
Is there a rule for spaces in between the `(` `)` and the closest token inside them?

Reply:
There is.  There's a rule for pretty much everything :)

Reply:
😄 -- cool, cause now that I saw those lines, they look bad.

How hard was to create the huge .sln and auto-apply the code fixes?

Reply:
> cool, cause now that I saw those lines, they look bad.

:smile:

> How hard was to create the huge .sln and auto-apply the code fixes?

Creating a hacky .sln was pretty easy.  I could check it in if desirable, though I expect it'll get out of date relatively quickly.  The less ideal part is it's a super gigantic .sln, and it quickly hits up against the limits of VS's "fix all" functionality.  @sharwell helped me workaround some of them.  In other cases I hit issues that I manually worked around.

Reply:
I see. Is it possible to do this through a Console App, that it opens every solution in corefx and auto-applies the code fixes in all of them?

Reply:
> Is it possible to do this through a Console App, that it opens every solution in corefx and auto-applies the code fixes in all of them?

The dotnet-format tool is supposed to do that, at least eventually, but I don't think it's "there" yet.  @sharwell can speak to that.